guides/choosing_mono_or_multi_repo.md

```diff
@@ -0,0 +1,96 @@
+
+Size of downloading the repository is also a consideration. A mono repo can be large and take a long time to download, particularly for new team members. A multi repo can be smaller and quicker to download, as each repository contains a smaller amount of code.
+
+Performance of scanning tools such as secret detection tools is also increased with smaller repositories.
+
+### Technology choices
+
+Mono repos are best suited for projects that use a single technology stack. Multi repos are best suited for projects that use multiple technology stacks.
+
+If a project uses multiple technology stacks, then it is more difficult to keep a single repository structure consistent.
+
+Multiple technologies in a single repository can also lead to conflicts between any plugins or dependencies that are used by the different technologies within the repository.
+
+### Collaboration
+
+Mono repos simplify collaboration as all the code is in one place.  This reduces the complexity of access control and visibility of code, pull requests, issues and release notes.
+
+However mono repos may also tightly coupled collaboration needs across independent teams.  This can lead to conflicts and bottlenecks in the development process.
+
+Multi repos are more difficult to collaborate as data is distributed across multiple repositories.  This can mean key data is missed which leads to risk.
+
+### CI/CD
+
+Mono repos simplify CI/CD as multiple projects can be built and tested together. This can reduce the complexity of creating and managing multiple pipelines and dependencies.
+
+However, mono repos can also lead to longer build times and increased complexity in managing dependencies, particularly if multiple technology choices are used.
+
+Additional tooling and complexity may be required to independently build and test projects within a mono repo depending on the scale.
+
+Multi repos require more configuration for CI/CD as each project needs its own pipeline which can lead to duplication of effort and complexity in co-ordination of pipeline updates.
+
+The advantage is a more optimised pipeline for each project.
+
+### Release process
+
+Mono repos are best suited for projects that require all code to be released together.
+
+Multi repos are best suited for projects that require independent deployments.
+
+The scale of the service should again be considered as releasing or rolling back a large scale mono repo code base can add unnecessary risk and complexity.
+
+### Commit history
+
+Mono repos have a single commit history which can make it easier to track changes and understand the history of the codebase.
+
+Multi repos have multiple commit histories which can make it more difficult to track changes and understand the history of the codebase, however the history is more focused.
+
+### Versioning
```

Comment:
I'd suggest maybe combining versioning and releasing as these two sections feel like they're making very similar points.

Reply:
Done.